tutorials/source-modeling/plot_freesurfer_coordinate_frames.py

```diff
@@ -0,0 +1,201 @@
+            path_effects.Normal()])
+    # reorient for our data orientation and label
+    ax.set(xlim=[0, 255], ylim=[255, 0], xlabel='k', ylabel='j',
+           title='i={}'.format(i))
+    fig.suptitle(suptitle)
+    fig.subplots_adjust(0.1, 0.1, 0.95, 0.85)
+
+
+imshow_mri(data, vox, dict(RAS=xyz_ras), 'MRI slice')
+
+###############################################################################
+# Notice that the axis scales (``i``, ``j``, and ``k``) are still in voxels
+# (ranging from 0-255); it's only the annotation text that we've translated
+# into real-world RAS in millimeters.
+#
+#
+# MRI coordinates in MNE-Python are really "Freesurfer surface RAS"
+# -----------------------------------------------------------------
+#
+# While :mod:`nibabel` uses standard RAS ``(x, y, z)`` coordinates, Freesurfer
+# uses *MRI surface RAS* coordinates; the transform from voxels to Freesurfer
+# MRI surface RAS coordinate frame is known in the `Freesurfer documentation
+# <https://surfer.nmr.mgh.harvard.edu/fswiki/CoordinateSystems>`_ as ``Torig``,
+# and in nibabel as :meth:`vox2ras_tkr
+# <nibabel.freesurfer.mghformat.MGHHeader.get_vox2ras_tkr>`.
+# Since MNE-Python uses Freesurfer extensively for surface computations (e.g.,
+# white matter, inner/outer skull meshes), internally MNE-Python often uses the
+# Freesurfer surface RAS coordinate system (not the :mod:`nibabel` RAS system).
+# We can do similar computations as before to figure out where the point is in
+# Freesurfer MRI coordinates:
+
+Torig = t1.header.get_vox2ras_tkr()
+print(t1.affine)
+print(Torig)
+xyz_mri = apply_trans(Torig, vox)
+imshow_mri(data, vox, dict(MRI=xyz_mri), 'MRI slice')
+
+###############################################################################
+# Knowing these relationships and being mindful about transformations, we
+# can get from a point in any given space to any other space. Let's start out
+# by plotting the Nasion on a saggital MRI slice:
+
+fiducials = mne.coreg.get_mni_fiducials('sample', subjects_dir=subjects_dir)
+nasion_mri = [d for d in fiducials if d['ident'] == FIFF.FIFFV_POINT_NASION][0]
+print(nasion_mri)  # note it's in MRI coords
+
+###############################################################################
+# When we print the nasion, it displays as a ``DigPoint`` and shows its
+# coordinates in millimeters, but beware that the underlying data is
+# :ref:`actually stored in meters <units>`,
+# so before transforming and plotting we'll convert to millimeters:
+
+nasion_mri = nasion_mri['r'] * 1000  # meters → millimeters
+nasion_vox = np.round(
+    apply_trans(np.linalg.inv(Torig), nasion_mri)).astype(int)
+imshow_mri(data, nasion_vox, dict(MRI=nasion_mri),
+           'Nasion estimated from MNI transform')
+
+###############################################################################
+# We can also take the digitization point from the MEG data, which is in the
+# "head" coordinate frame:
```

Comment:
Link to https://mne.tools/stable/overview/implementation.html#the-head-and-device-coordinate-systems?